compiler/rustc_span/src/lib.rs

```diff
@@ -1065,33 +1065,41 @@ impl Span {
 
     #[inline]
     pub fn remove_mark(&mut self) -> ExpnId {
-        let mut span = self.data();
-        let mark = span.ctxt.remove_mark();
-        *self = Span::new(span.lo, span.hi, span.ctxt, span.parent);
+        let mut mark = ExpnId::root();
```

Comment:
I guess this one is intentionally different from the others (which start as `None`), may it be documented?

Reply:
Not sure what to document, these functions don't have their own logic and just forward to the underlying `SyntaxContext` methods, which may have different signatures and have their own documentation.

Reply:
ahhh indeed, apologies, I didn't look at the the function signature